groups/managers.py

```diff
@@ -1,8 +1,19 @@
+import datetime
+
+from django.apps import apps
 from django.contrib.auth import get_user_model
 from django.db import models
 from polymorphic import PolymorphicManager, PolymorphicQuerySet
 
 
+DEFAULT_ACTIVE_THRESHOLD = apps.get_app_config('groups').default_active_threshold_days
+
+
+def get_threshold_date(threshold_days=DEFAULT_ACTIVE_THRESHOLD):
+    """Return the earliest posting date a comment can have and still be recent."""
+    return datetime.date.today() - datetime.timedelta(days=threshold_days)
```

Comment:
The problem with that approach is it doesn't work if we need to work with multiple timezones, which seems quite likely for international websites.